corehq/apps/cloudcare/static/cloudcare/js/formplayer/spec/case_list_pagination_spec.js

```diff
@@ -2,7 +2,7 @@ hqDefine("cloudcare/js/formplayer/spec/case_list_pagination_spec", function () {
     describe('#paginateOptions', function () {
         var paginateItems = hqImport("cloudcare/js/formplayer/utils/utils");
         it('Should return paginateOptions', function () {
-            var case1 = paginateItems.paginateOptions(0, 15);
+            var case1 = paginateItems.paginateOptions(0, 15, 3);
```

Comment:
Should there be `showPagination` assertions for `case1` and `case4` in this block as well?

Reply:
Meh. It wouldn't test additional behavior.